final/numbers.md

```diff
@@ -82,7 +82,7 @@ num.toPrecision(4); // '391.6'
 num.toPrecision(5); // '391.59'
 ```
 
-####[isInteger()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger)
+####[Number.isInteger()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger)
```

Comment:
These are meant to be methods on the Number object. I suppose it could go either way, but I think I prefer to leave it out on the methods, to align with the rest of the posts and minimize redundancy.

Reply:
`isInteger` seems to be a method on the Number constructor (`Number.isInteger(number)`) whereas the other methods are instance methods of the particular number object (eg. `num.toExponential()`)